project/Dependencies.scala

```diff
@@ -145,6 +147,10 @@ object Dependencies {
     "com.github.ghik" %% "silencer-lib" % Version.silencer % Provided cross CrossVersion.full
   )
 
+  private val load: Seq[ModuleID] = Seq(
+    "com.github.pureconfig" %% "pureconfig" % Version.pureConfig
+  )
```

Comment:
Why not to make this a single dependency `pureConfig` and then..

Reply:
done